cfripper/cli.py

```diff
@@ -0,0 +1,135 @@
+import json
+import logging
+import sys
+from io import TextIOWrapper
+from pathlib import Path
+from typing import Dict, Optional
+
+import click
+import pycfmodel
+from pycfmodel.model.cf_model import CFModel
+
+from cfripper.__version__ import __version__
+from cfripper.config.config import Config
+from cfripper.model.result import Result
+from cfripper.model.utils import convert_json_or_yaml_to_dict
+from cfripper.rule_processor import RuleProcessor
+from cfripper.rules import DEFAULT_RULES
+
+
+def setup_logging(level: str) -> None:
+    logging.basicConfig(level=logging._nameToLevel[level], format="%(message)s")
+
+
+def get_cfmodel(file: TextIOWrapper) -> CFModel:
+    template = convert_json_or_yaml_to_dict(file.read())
+    cfmodel = pycfmodel.parse(template)
+    return cfmodel
+
+
+def format_result(result: Result, format: str) -> str:
+    if format == "json":
```

Comment:
split in 2 functions: format_result_json and format_result_yaml

Reply:
Actually, it would be `format_result_json` and `format_result_txt`